### PR TITLE
Update main.yaml

### PR DESCRIPTION
**Pull request by vdua**
moving circle ci config to github actions

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
https://main--afb--adobe.hlx.live/demo/claim

### CODE REVIEW

**Comment by gargadobe** [comment]
@vdua Please delete the .circleci folder in same pr as it is replacement of same

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ Please always provide the [GitHub issue(s)](../issues) your PR is for, as well a
 Fix #<gh-issue-id>
 
 Test URLs:
```
Comment (gargadobe) [inline .github/pull_request_template.md:5]: it is template only. the values are getting replaced here https://github.com/adobe-rnd/aem-boilerplate-forms/blob/main/.github/workflows/cleanup-on-create.yaml#L53 when someone create repo.

**Review by gargadobe — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (vdua) [reply .github/pull_request_template.md:5]: We need the template for this repo as well.

Comment (gargadobe) [reply .github/pull_request_template.md:5]: then we need to update README not template ?

Comment (vdua) [reply .github/pull_request_template.md:5]: When you will raise a PR here, it should show `https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/`  in the PR.

**Review by gargadobe — DISMISSED** [review]

**Pull request merged**